tools/update_repo.sh

```diff
@@ -657,7 +657,7 @@ function pack_rpm {
     done
 
     # create the new repository metadata files
-    createrepo --no-database --update --workers=2 \
+    createrepo_c --no-database --update --workers=2 \
```

Comment:
Do you looked over the [list of changes](https://github.com/rpm-software-management/createrepo_c#differences-in-behavior-between-createrepo_c-and-createrepo) to spot possible negative consequences?

Reply:
Right, I've checked it, but main confidence that it works correct, was after I've checked that all packages successfully saved and were available to install from temporary S3 YUM repositories. Check the following branch test workflows:
https://github.com/tarantool/tarantool/tree/avtikhon/gh-qa-110-ci-deb-tmp
and S3 repository:
s3://tarantool_repo/check_live